packages/client-gear-playground/webpack.config.babel.js

```diff
@@ -12,81 +13,125 @@ module.exports = {
     publicPath: '/build/',
   },
   resolve: {
-    extensions: ['', '.js', '.json', '.jsx'],
-    root: [
+    extensions: ['.js', '.json', '.jsx'],
+    modules: [
       __dirname,
       path.resolve(__dirname),
+      path.resolve(__dirname, 'node_modules'),
+      path.resolve(__dirname, '../kununu-footer/node_modules'),
+      path.resolve(__dirname, '../kununu-header/node_modules'),
+      path.resolve(__dirname, '../kununu-logo/node_modules'),
```

Comment:
It should work with just 
```
path.resolve(__dirname),
'node_modules
```
I don't think you need to specify each package like this. It will also be bigger and harder to maintain the more packages we add.